Parsers/ASimAuditEvent/Parsers/vimAuditEventInfobloxBloxOne.yaml

```diff
@@ -0,0 +1,159 @@
+Parser:
+  Title: AuditEvent ASIM parser for Infoblox BloxOne
+  Version: '0.1.0'
+  LastUpdated: Jun 26 2024
+Product:
+  Name: Infoblox BloxOne
+Normalization:
+  Schema: AuditEvent
+  Version: '0.1'
+References:
+- Title: ASIM AuditEvent Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Infoblox BloxOne Documentation
+  Link: https://docs.infoblox.com/space/BloxOneThreatDefense
+Description: |
+  This ASIM parser supports normalizing AuditEvent logs from Infoblox BloxOne to the ASIM AuditEvent normalized schema. These events are captured through the Azure Monitor Agent (AMA) which are sent by the Data Connector Service of Infoblox BloxOne.
+ParserName: vimAuditEventInfbloxBloxOne
+EquivalentBuiltInParser: _Im_AuditEvent_InfobloxBloxOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: srcipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: operation_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventtype_in
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventresult
+    Type: string
+    Default: '*'
+  - Name: actorusername_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: object_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: newvalue_has_any
+    Type: dynamic
+    Default: dynamic([])
+ParserQuery: |
+  let EventSeverityLookup = datatable (LogSeverity:string, EventSeverity:string)
+      [
+      "0", "Low",
+      "1", "Low",
+      "2", "Low",
+      "3", "Low",
+      "4", "Medium",
+      "5", "Medium",
+      "6", "Medium",
+      "7", "High",
+      "8", "High",
+      "9", "High",
+      "10", "High"
+  ];
+  let parser = (disabled: bool = false, starttime: datetime=datetime(null), endtime: datetime=datetime(null), eventresult: string='*', operation_has_any: dynamic=dynamic([]), eventtype_in: dynamic=dynamic([]), srcipaddr_has_any_prefix: dynamic=dynamic([]), actorusername_has_any: dynamic=dynamic([]), object_has_any: dynamic=dynamic([]), newvalue_has_any: dynamic=dynamic([])) {
+      CommonSecurityLog
+      | where not(disabled) 
+          and (isnull(starttime) or TimeGenerated >= starttime) 
+          and (isnull(endtime) or TimeGenerated <= endtime) 
+          and DeviceVendor == "Infoblox"
+          and DeviceEventClassID has "AUDIT"
+          and (eventresult == "*" or EventOutcome =~ eventresult)
```

Comment:
Does EventOutcome always going to have same supported values as ASim for EventResult?
Success, Partial, Failure, NA (Not Applicable).

Reply:
We are currently unable to find a comprehensive list of possible values for the EventOutcome field. However, we have generated data with values that are supported by ASIM for the EventResult field.